cypress/integration/application/index.js

```diff
@@ -34,6 +36,17 @@ describe('Application', () => {
             type: 'application/json',
           });
         });
+        it(`should awaken fresh data for pages for later tests`, () => {
+          // Add more here if you want to awaken fresh data for other page types
+          if (serviceHasPageType(service, 'topicPage')) {
+            const paths = getPaths(service, 'topicPage');
+            paths.forEach(currentPath => {
+              const fullPath = `${envConfig.baseUrl}${currentPath}`;
```

Comment:
Maybe more readable as

`const fullPath = envConfig.baseUrl + currentPath;`

If linting allows?

Reply:
Don't think linting allows - would allow [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat) though

Reply:
Is it better just for readability reasons?